tests/ulyanov_et_al_2016/test_modules.py

```diff
@@ -120,3 +167,217 @@ def test_get_activation_module(subtests):
             if isinstance(norm_module, nn.LeakyReLU):
                 with subtests.test("slope"):
                     assert norm_module.negative_slope == pytest.approx(0.01)
+
+
+def test_UlyanovEtAl2016ConvBlock(subtests):
+    in_channels = out_channels = 3
+    kernel_size = 3
+    stride = 1
+    padding = (1, 1, 1, 1)
+    conv_block = modules.UlyanovEtAl2016ConvBlock(
+        in_channels, out_channels, kernel_size, stride=stride
+    )
+
+    assert isinstance(type(conv_block), type(modules.SequentialWithOutChannels))
+
+    with subtests.test("modules"):
+        assert len(conv_block) == 4
+        assert isinstance(type(conv_block[0]), type(nn.ReflectionPad2d))
```

Comment:
See discussion in https://github.com/pmeier/pystiche_papers/pull/82#pullrequestreview-458947475

```suggestion
        assert isinstance(conv_block[0], nn.ReflectionPad2d)
```